src/utils/paths.js

```diff
@@ -112,6 +112,16 @@ export class PathConditionsImplementation extends PathConditions {
       savedBaseConditions.refineBaseConditons(realm);
     }
   }
+
+  clone(): PathConditionsImplementation {
```

Comment:
@hermanventer says we shouldn't have to clone, is that right?

Reply:
Maybe not clone, but wrap the realm path conditions in a new path conditions object. In any case, watch this PR: #2538

Reply:
Do not clone.